tests/model_forms/models.py

```diff
@@ -212,7 +212,9 @@ def __str__(self):
 
 
 class Price(models.Model):
-    price = models.DecimalField(max_digits=10, decimal_places=2)
+    # Numeric field/Decimal Field is not supported in primary key/unique key.
+    # So changed it to Float field.
+    price = models.FloatField()
```

Comment:
We may be asking for trouble continuing to patch tests on this fork instead of rewriting them in spanner-django.

Reply:
model migrations are applied by django test at the very start, so even if we skip a particular test this model migration causes a failure. Either we skip the entire test module or we do this overwrite, we don't have very good options here unit the feature is supported by Spanner:(